pycqed/instrument_drivers/meta_instrument/LutMans/mw_lutman.py

```diff
@@ -104,13 +244,27 @@ def _add_waveform_parameters(self):
         self.spec_func = wf.block_pulse
 
         self._add_channel_params()
+        # FIXME: Remove after the end of RUS experiment
```

Comment:
Parameters added from line 247-260 should be removed. These were introduced by mistake and constitute a bug